$OpenBSD: patch-rss2email_feed_py,v 1.1 2021/06/28 07:24:00 sthen Exp $

Maintain compatibility with feedparser 5, which doesn't have the feedparser.http module.

Index: rss2email/feed.py
--- rss2email/feed.py.orig
+++ rss2email/feed.py
@@ -445,7 +445,7 @@ class Feed (object):
         elif isinstance(exc, _SOCKET_ERRORS):
             _LOG.error('{}: {}'.format(exc, self))
             warned = True
-        elif isinstance(exc, _feedparser.http.zlib.error):
+        elif isinstance(exc, _feedparser.zlib.error):
             _LOG.error('broken compression: {}'.format(self))
             warned = True
         elif isinstance(exc, (IOError, AttributeError)):
